src/Shared/AssemblyNameExtension.cs

```diff
@@ -734,29 +734,13 @@ private bool EqualsImpl(AssemblyNameExtension that, bool ignoreVersion, bool con
         /// <summary>
         /// Allows the comparison of the culture.
         /// </summary>
-        internal bool CompareCulture(AssemblyNameExtension that)
+        internal static bool CompareCultures(AssemblyName a, AssemblyName b)
         {
             // Do the Cultures match?
 #if FEATURE_ASSEMBLYNAME_CULTUREINFO
-            CultureInfo aCulture = CultureInfo;
-            CultureInfo bCulture = that.CultureInfo;
-            if (aCulture == null)
-            {
-                aCulture = CultureInfo.InvariantCulture;
-            }
-            if (bCulture == null)
-            {
-                bCulture = CultureInfo.InvariantCulture;
-            }
-
-            if (aCulture.LCID != bCulture.LCID)
-            {
-                return false;
-            }
-
-            return true;
+            return a?.CultureInfo?.LCID == b?.CultureInfo?.LCID;
```

Comment:
This isn't the same comparison as before - if one is Invariant and one is null, this used to return true, now it returns false.

Reply:
You're right. I made the .net core implementation follow the full framework one and not the other way around